docs/recipes/af3.md

```diff
@@ -0,0 +1,187 @@
+at `/orcd/datasets/001/alphafold3`.
+
+Once you have everything you need, you will be ready to run AlphaFold 3. We will
+now go through a test case adapted from the
+[AlphaFold 3 GitHub Repository](https://github.com/google-deepmind/alphafold3).
+From the working directory, create an output directory and a test input file:
+
+```bash
+mkdir af_output
+mkdir af_input
+vi af_input/fold_input.json
+```
+
+```title="fold_input.json"
+{
+  "name": "2PV7",
+  "sequences": [
+    {
+      "protein": {
+        "id": ["A", "B"],
+        "sequence": "GMRESYANENQFGFKTINSDIHKIVIVGGYGKLGGLFARYLRASGYPISILDREDWAVAESILANADVVIVSVPINLTLETIERLKPYLTENMLLADLTSVKREPLAKMLEVHTGAVLGLHPMFGADIASMAKQVVVRCDGRFPERYEWLLEQIQIWGAKIYQTNATEHDHNMTYIQALRHFSTFANGLHLSKQPINLANLLALSSPIYRLELAMIGRLFAQDAELYADIIMDKSENLAVIETLKQTYDEALTFFENNDRQGFIDAFHKVRDWFGDYSEQFLKESRQLLQQANDLKQG"
+      }
+    }
+  ],
+  "modelSeeds": [1],
+  "dialect": "alphafold3",
+  "version": 1
+}
+```
+
+You can either run this in an interactive session or in a batch job. If you have
+access to a partition with a GPU, replace the partition name as necessary.
+
+=== "Interactive"
+
+    Request an interactive session with a GPU:
+
+    ```bash
+    salloc -N 1 -n 16 -p mit_normal_gpu --gres=gpu:1
+    ```
+
+    Run this script:
+
+    ```bash title="run_alphafold.sh"
+    #!/bin/bash
+ 
+    module load apptainer
+
+    # Enter the path to the AF3 dataset:
+    DATABASES_DIR=/orcd/datasets/001/alphafold3
+
+    # Enter the directory of the AF3 material:
+    WORKDIR=~/af3
+
+    apptainer exec \
+        --bind $WORKDIR/af_input:/root/af_input \
+        --bind $WORKDIR/af_output:/root/af_output \
+        --bind $WORKDIR/models:/root/models \
+        --bind $WORKDIR/alphafold3:/root/alphafold3 \
+        --bind $DATABASES_DIR:/root/public_databases \
+        --nv \
+        alphafold3.sif \
+        python /root/alphafold3/run_alphafold.py \
+        --json_path=/root/af_input/fold_input.json \
+        --model_dir=/root/models \
+        --output_dir=/root/af_output \
+        --db_dir=/root/public_databases
+    ```
+
+=== "Batch"
+
+    Create your batch job script:
+
+    ```bash title="run_alphafold.sbatch"
+    #!/bin/bash
+ 
+    #SBATCH -N 1
+    #SBATCH -n 16
+    #SBATCH -p mit_normal_gpu
+    #SBATCH --gres=gpu:1
+
+    module load apptainer
+
+    # Enter the path to the AF3 dataset:
+    DATABASES_DIR=/orcd/datasets/001/alphafold3
+
+    # Enter the directory of the AF3 material:
+    WORKDIR=~/af3
+
+    apptainer exec \
+        --bind $WORKDIR/af_input:/root/af_input \
+        --bind $WORKDIR/af_output:/root/af_output \
+        --bind $WORKDIR/models:/root/models \
+        --bind $WORKDIR/alphafold3:/root/alphafold3 \
+        --bind $DATABASES_DIR:/root/public_databases \
+        --nv \
+        alphafold3.sif \
+        python /root/alphafold3/run_alphafold.py \
+        --json_path=/root/af_input/fold_input.json \
+        --model_dir=/root/models \
+        --output_dir=/root/af_output \
```

Comment:
Can you add a note after this script that shows where to find the output when the job is done?